web/src/engine/main/src/keymanEngine.ts

```diff
@@ -202,6 +202,8 @@ export default class KeymanEngine<
 
     this.contextManager.configure({
       resetContext: (target) => {
+        // Could reset the target's deadkeys here, but it's really more of a 'core' task.
+        // So we delegate that to keyboard-processor.
```

Comment:
Considering that deadkeys are _very much_ managed by desktop-Core, this seemed like the right decision... given that we plan to integrate desktop-Core in the future.